### PR TITLE
Dijkstra's Algorithm

### PR DESCRIPTION
**Pull request by snehalkhandve**
I have added Dijkstra's Algorithm for finding shortest path between single source and multiple destinations in a given graph.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
Please resolve conflicts

**Comment by snehalkhandve** [comment]
Yes resolved the conflicts

**Comment by Bhavnaharitsa** [comment]
> Yes resolved the conflicts

Still not resolved

File: Dijkstra's_Algorithm/Dijkstra's_Algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+5
+
+Undirected weighted graph
+
+Enter all edges one by one 
+Format for input : vertex1 vertex2 weight
+1 2 2
+1 3 10
+1 4 4
+2 3 2
```
Comment (Bhavnaharitsa) [inline Dijkstra's_Algorithm/Dijkstra's_Algorithm.cpp]: put this comment on top

File: Dijkstra's_Algorithm/Dijkstra's_Algorithm.cpp [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+				queue.insert(make_pair(dist[to], to));
+			}
+		}	
+	}
+	return dist;
+}
+
+int main() {
+	cout<<"\nEnter number of vertices\n";
+	int n; cin >> n;
```
Comment (Bhavnaharitsa) [inline Dijkstra's_Algorithm/Dijkstra's_Algorithm.cpp]: don;t use single letter word...please give it a sensible name stating it's purpose

**Comment by snehalkhandve** [comment]
Really sorry for my ignorance .I had to close this pull request due to branch issues . I have opened another pull request #62  Pleaseeeee review that pull request @Bhavnaharitsa

**Pull request closed without merging**